packages/netsuite-adapter/test/change_validators/extra_reference_dependencies.test.ts

```diff
@@ -0,0 +1,133 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import { ElemID, InstanceElement, ObjectType, ReferenceExpression, toChange } from '@salto-io/adapter-api'
+import { fileType } from '../../src/types/file_cabinet_types'
+import { customsegmentType } from '../../src/autogen/types/standard_types/customsegment'
+import extraReferenceDependenciesValidator from '../../src/change_validators/extra_reference_dependencies'
+import { CUSTOM_RECORD_TYPE, METADATA_TYPE, NETSUITE, PATH, SCRIPT_ID } from '../../src/constants'
+
+
+describe('extra reference changes', () => {
```

Comment:
to match the validator name
```suggestion
describe('extra reference dependencies', () => {
```